app/assets/javascripts/initializers/initializeBaseTracking.js.erb

```diff
@@ -227,6 +227,8 @@ function trackAdImpression(token, adBox) {
 }
 
 function trackAdClick(token, clickedElement) {
+  console.log("CLICKED!", clickedElement);
```

Comment:
I think maybe some WIP code has sneaked its way in with the latest commit?

Reply:
Ugh, darnit - thank you for catching that. 🙈